packages/worker/src/worker-options.ts

```diff
@@ -487,14 +510,17 @@ export function appendDefaultInterceptors(
 export function addDefaultWorkerOptions(options: WorkerOptions): WorkerOptionsWithDefaults {
   const { maxCachedWorkflows, showStackTraceSources, namespace, reuseV8Context, ...rest } = options;
   const debugMode = options.debugMode || isSet(process.env.TEMPORAL_DEBUG);
+  const maxConcurrentWorkflowTaskExecutions = options.maxConcurrentWorkflowTaskExecutions ?? 100;
+  const maxConcurrentActivityTaskExecutions = options.maxConcurrentActivityTaskExecutions ?? 100;
+
   return {
     namespace: namespace ?? 'default',
     identity: `${process.pid}@${os.hostname()}`,
     shutdownGraceTime: 0,
-    maxConcurrentActivityTaskExecutions: 100,
     maxConcurrentLocalActivityExecutions: 100,
     enableNonLocalActivities: true,
-    maxConcurrentWorkflowTaskExecutions: 100,
+    maxConcurrentWorkflowTaskPolls: Math.min(5, maxConcurrentWorkflowTaskExecutions),
```

Comment:
Users should be able to configure these

Reply:
If you mean that `maxConcurrentWorkflowTaskPolls` should be user-configurable, they already can. This line sets the default value. `...rest` below will overwrite that default value with the user's value if present. After that, `compileWorkerOptions` will warn if the value is invalid (which should only be possible if value was provided by the user).